src/odemis/model/_metadata.py

```diff
@@ -108,6 +112,14 @@
 MD_POL_RHC = "rhc"  # (string) polarization analyzer position
 MD_POL_LHC = "lhc"  # (string) polarization analyzer position
 
+MD_STREAK_TIMERANGE = "Streak Time Range"  # Time range for one streak/sweep
+MD_STREAK_MCPGAIN  = "Streak MCP Gain"  # Multiplying gain for microchannel plate
+MD_STREAK_MODE = "Streak Mode"  # Mode of streak camera (Focus (Off) or Operate (On))
+
+MD_DELAY_TRIGGER = "Trigger Delay"  # Trigger delay between ext. trigger and starting of the streak/sweeping
+MD_DELAY_REPRATE = "Repetition Rate"  # Repetition Rate of the trigger signal
```

Comment:
Let's call this metadata "MD_TRIGGER_RATE". Or alternatively, you could make it a period in second, instead of being a in Hz, as the rest of the metadata related to time. In such case, it could be called "MD_TRIGGER_PERIOD".